src/test/SemanticVersion_test.cpp

```diff
@@ -0,0 +1,260 @@
+//------------------------------------------------------------------------------
```

Comment:
I think this (and all other new files) are using tabs for indent. I think CodingStyle.md says we use 4 spaces, no tabs.